### PR TITLE
add the company logos

### LINKED ISSUE #668: Add current sponsor logos

### PR DESCRIPTION
**Pull request by dquach217**
Fixes #668

### CODE REVIEW

**Comment by dquach217** [comment]
@sman591 could you help me with the last level of logos? Chris wanted me to have the last two logos side by side. It is proving to be difficult and was wondering if you could fix that portion

**Comment by sman591** [comment]
Sure I'll take a look right after I fix this one bug with emails

**Comment by sman591** [comment]
Updated: 
screenshot 2018-11-25 23 50 10

File: app/views/pages/index.html.haml [hunk 1]
```diff
@@ -117,7 +117,23 @@
...
-    %h2.text-center Sponsors
+    .sponsors
+      %h2.text-center Sponsors
+      .sponsors-col.sponsors-col-x-lg
+        %a.sponsor{ href: 'https://www.datto.com', title: 'Datto', target: 'blank' }
+          = image_tag 'sponsors/datto.png'
+      .sponsors-col.sponsors-col-lg
+        %a.sponsor{ href: 'https://cloud.google.com', title: 'Google Cloud Platform', target: 'blank' }
+          = image_tag 'sponsors/google_cloud_platform.png'
+      .sponsors-col.sponsors-col-med
```
Comment (sman591) [inline app/views/pages/index.html.haml]: Should these be "row" instead of "col"?

**Pull request merged**